services/scan/src/precinct_scanner_app.test.ts

```diff
@@ -859,17 +859,20 @@ test('insert second ballot while first ballot is returning', async () => {
 });
 
 test('jam on scan', async () => {
-  const { app, mockPlustek } = await createApp();
+  const { app, mockPlustek } = await createApp({
+    DELAY_RECONNECT_ON_UNEXPECTED_ERROR: 500,
+  });
   await configureApp(app);
 
   await mockPlustek.simulateLoadSheet(ballotImages.completeBmd);
   await waitForStatus(app, { state: 'ready_to_scan' });
 
   mockPlustek.simulateJamOnNextOperation();
   await post(app, '/scanner/scan');
-  await waitForStatus(app, { state: 'jammed' });
-
-  await mockPlustek.simulateRemoveSheet();
+  await waitForStatus(app, {
+    state: 'recovering_from_error',
+    error: 'plustek_error',
+  });
```

Comment:
If we get a jam error returned from the scan command, we treat it as an unexpected error now. I tried to reproduce this manually and couldn't do it - I think scan only returns a jam error after a long time trying to ingest the paper, but we now handle that with a timeout. So this case really would be "unexpected" now.